src/features/Students/StudentsTable/columns.jsx

```diff
@@ -30,54 +33,30 @@ const getColumns = props => [
     ),
   },
   {
-    Header: 'Created',
-    accessor: 'created',
-    Cell: ({ row }) => (
-      row.values.created
-        ? new Date(row.values.created).toUTCString()
-        : ''
-    ),
-  },
-  {
-    Header: 'First Access',
-    accessor: 'firstAccess',
-    Cell: ({ row }) => (
-      row.values.firstAccess
-        ? new Date(row.values.firstAccess).toUTCString()
-        : ''
-    ),
-  },
-  {
-    Header: 'Last Access',
-    accessor: 'lastAccess',
-    Cell: ({ row }) => (
-      row.values.lastAccess
-        ? new Date(row.values.lastAccess).toUTCString()
-        : ''
-    ),
-  },
-  {
-    Header: 'Grade',
-    accessor: 'grade',
-    Cell: ({ row }) => <Badge variant={row.values.grade ? 'success' : 'danger'}>{row.values.grade ? 'pass' : 'fail'}</Badge>,
-  },
-  {
-    Header: 'Action',
+    Header: 'Status',
     accessor: 'status',
-    disableSortBy: true,
     Cell: ({ row }) => {
-      const value = row.values;
-
-      if (value.status !== 'Pending') {
-        return null;
+      switch (row.values.status) {
+        case 'Active':
+          return <Badge variant="success" light>Active</Badge>;
+        case 'Inactive':
+          return <Badge variant="secondary" light>Inactive</Badge>;
+        case 'Expired':
+          return <Badge variant="danger" light>Expired</Badge>;
+        case 'Pending':
+          return <Badge variant="warning" light>Pending</Badge>;
+        default:
+          return null;
       }
-
-      return <Button variant="outline-danger" onClick={() => { props.openAlertModal(); props.setRow(value); }}>Revoke</Button>;
     },
   },
+  {
+    Header: 'Exam Ready',
+    accessor: 'examReady',
```

Comment:
```suggestion
    accessor: 'examReady',
    Cell: ({ row }) => {
      if (row.values.examReady === true) {
        return 'yes'
      }

      return 'no'
    },
```


There's probably a better way to do it. Otherwise the table looks good using the course operations endpoint.

![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/73fbf3e2-6bbb-4660-8d84-7092e74d9629)

Reply:
Sorry, that's the old code, I will change that

Reply:
@anfbermudezme fixed